runhouse/servers/cluster_servlet.py

```diff
@@ -91,6 +93,11 @@ async def __init__(
         # will be used when self.gpu_metrics will be updated by different threads.
         self.lock = threading.Lock()
 
+        # will be used in the periodic loop which sends cluster logs to den. Since we are saving logs to the server.log
+        # file asynchronously, there might be a use case where we want to send logs to den but the server.log has not
+        # been created yet, since it is a freshly initialized cluster servlet. In this case, we will return an empty string.
+        self.is_freshly_initialized = True
```

Comment:
maybe `is_log_file_ready`? freshly initialized was a little unclear to me at first